BUILDING.md

```diff
@@ -181,9 +183,9 @@ In case you preferred to install and use the standalone **CMake** tool:
 
 While still in the project root:
 
-1) `cd .. && mkdir --parents rpcs3_build && cd rpcs3_build`
-2) `cmake ../rpcs3/ && make` or `CXX=g++-13 CC=gcc-13 cmake ../rpcs3/ && make` to force these compilers
-3) run RPCS3 with `./bin/rpcs3`
+1) `cmake -B build -G Ninja` to generate build files
+2) `cmake --build build` to compile with Clang (default), or `CXX=g++-13 CC=gcc-13 cmake --build build` to compile with GCC
```

Comment:
I'm not sure about the GCC part, doesn't it involve `CMAKE_C_COMPILER` and `CMAKE_CXX_COMPILER`? I'll try to find some time to test.